src/http/register-websocket/index.js

```diff
@@ -4,15 +4,17 @@ let _connection = require('./connection')
 let send = require('./send')
 
 module.exports = function registerWebSocket (params) {
-  let { app, cwd, httpServer, inventory, update } = params
+  let { app, cwd, httpServer, inventory, update, httpPort } = params
 
   let wss = new WebSocket.Server({ noServer: true })
-
+  let connectedAt = Date.now()
+  let domainName = `localhost:${httpPort}`
+  let stage = 'staging'
```

Comment:
`stage` should be `testing`, or perhaps more appropriately: `process.env.NODE_ENV || 'testing'`